### DISCUSSION TITLE
How to use ComboBox in forms?

### DISCUSSION THREAD

**Discussion by samsnori** [adobe/react-spectrum#5711] [Q&A] [answered]
How do I get the value from a `ComboBox` inside a Form? 

When I use a `Switch` the value is automatically added to the form data. When using the `ComboBox` I do not get the selected items. I can use the `onSelectionChange` to keep track of the selected values. Should I use this instead?

If I must use `onSelectionChange` to keep track of the values, why is the behavior different from e.g. `Switch` ?

**Comment by devongovett** [comment, marked as answer]
See https://react-spectrum.adobe.com/react-aria/ComboBox.html#html-forms

**Reply by samsnori** [reply]
Thanks @devongovett I did read that page but overlooked that section.
